src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AssemblyBasedCallerUtilsUnitTest.java

```diff
@@ -64,7 +64,7 @@ public void testfinalizeRegion() {
         activeRegion.addAll(reads);
         SampleList sampleList = SampleList.singletonSampleList("tumor");
         Byte minbq = 9;
-        AssemblyBasedCallerUtils.finalizeRegion(activeRegion, false, false, minbq, header, sampleList, false, false);
+        AssemblyBasedCallerUtils.finalizeRegion(activeRegion, false, false, minbq, header, sampleList, true, false);
```

Comment:
Can you confirm that the modified version of this test fails without the `copy()` call you added above?

Reply:
I have already tested it. It does fail without the above fix. the problem is that when we added `correctOverlappingBaseQualities` as an argument to `finalizeRegion()` it was mistakenly set to false for this method, which means this test was doing nothing and thus we missed that the last round of refactoring broke the assertion in this test by allowing un-coppied base qualities to be adjusted.

Reply:
Can you add a comment then mentioning that that boolean argument is critical for the test to be meaningful?

Reply:
@droazen I added comments better explaining how the test works. Is this okay to merge?